develop/dev-guide-bi-looker-stduio.md

```diff
@@ -0,0 +1,128 @@
+
+## Import Dataset
+
+You can use the S&P 500 dataset provided as an Interactive Tutorial of TiDB Serverless. 
+
+<SimpleTab>
+<div label="TiDB Serverless">
+
+1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page, and click the question mark icon in the bottom-right corner. A Help dialog is displayed. 
+
+2. Click **Interactive Tutorials**. Then click **S&P 500 Analysis**.
+
+3. In the **Import S&P 500 Dataset** dialog, select your Serverless cluster. Then click the **Import Dataset** button. It will start importing briefly.
+
+4. After the status changed to **Imported**, you can close this dialog. Click the **Exit Tutorial** button in the bottom-left corner of the dialog. 
+
+</div>
+
+## Connect to TiDB
+
+1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page, and then click the name of your target cluster to go to its overview page.
+
+2. Click **Connect** in the upper-right corner. A connection dialog is displayed.
+
+3. Set **Operating System** to `Windows` in the connection dialog. Then Set **Connect With** to `General`. You can download [CA cert](https://letsencrypt.org/certs/isrgrootx1.pem) at the note below. 
+
+    > **Tip:**
+    >
+    > As TiDB Serverless requires a secure TLS connection between the client and the cluster, you need this CA cert for connection settings on Looker Studio.
+
+4. Click **Generate Password** to create a random password.
+
+    > **Tip:**
+    >
+    > If you have created a password before, use the original password or click **Reset Password** to generate a new one.
+
+5. Open Looker Studio and create a Blank Report. Select the' MySQL' connector in the **Add data to report** screen.
+
+6. In the setting pane, configure the following connection parameters:
+
+    - **Host Name or IP**: enter the `HOST` parameter from the TiDB Cloud connection dialog.
+    - **Port(Optional)**: enter the `PORT` parameter from the TiDB Cloud connection dialog.
+    - **Database**: enter the database you want to connect to. In this tutorial, set `sp500insight`.
+    - **Username**: enter the `USERNAME` parameter from the TiDB Cloud connection dialog.
+    - **Password**: enter the `PASSWORD` parameter from the TiDB Cloud connection dialog.
+    - Check **Enable SSL**.
+        - Click the icon on the right of **MySQL SSL Client Configuration Files**. Then, select the downloaded CA file in step 3. 
+
+    ![Looker Studio: configure connection settings for TiDB Serverless](/media/develop/looker-studio-configure-connection.png)
+
+7. Click **AUTHENTICATE**. 
+
+If the authentication succeeds, you can see tables in the database. 
+
+## Create a simple chart
+
+Now, you can use TiDB Serverless as a data source. Let's make a simple chart with data. 
+
+1. In the setting pane, select **CUSTOM QUERY**. 
+
+2. Put the SQL below, then click **Add** in the bottom-right corner. 
+    ```sql
```

Comment:
````suggestion

    ```sql
````

This might fix this:
```
[2024-01-11T07:11:19.119Z] develop/dev-guide-bi-looker-stduio.md: this file has manual line breaks in the following lines:
[2024-01-11T07:11:19.119Z] 
[2024-01-11T07:11:19.119Z] MANUAL LINE BREAKS: L99
```

Reply:
And this:

```
[2024-01-11T07:11:11.976Z] develop/dev-guide-bi-looker-stduio.md: 87: MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```sql"]
```